docs/usage.md

```diff
@@ -128,16 +128,19 @@ Let's call the source mysqld instance _donor_.
 We use [the clone plugin][CLONE] to copy the whole data quickly.
 After the cloning, MOCO needs to create some user accounts and install plugins.
 
-**On the donor**, you need to install the plugin and create two user accounts as follows:
+**On the donor**, you need to install the plugin and create two user accounts as follows[^note]:
 
 ```console
 mysql> INSTALL PLUGIN clone SONAME mysql_clone.so;
 mysql> CREATE USER 'clone-donor'@'%' IDENTIFIED BY 'xxxxxxxxxxx';
 mysql> GRANT BACKUP_ADMIN, REPLICATION SLAVE ON *.* TO 'clone-donor'@'%';
 mysql> CREATE USER 'clone-init'@'localhost' IDENTIFIED BY 'yyyyyyyyyyy';
 mysql> GRANT ALL ON *.* TO 'clone-init'@'localhost' WITH GRANT OPTION;
+mysql> GRANT PROXY ON ''@'' TO 'clone-init'@'localhost' WITH GRANT OPTION;
 ```
 
+[^note]: The line of `GRANT PROXY ...` has been added for moco-agent >= 0.6.9 in response to [cybozu-go/moco-agent#58](https://github.com/cybozu-go/moco-agent/pull/58).
```

Comment:
This section describes a setup for cloning an external mysqld which is not necessarily a moco cluster.
So, I guess this note is misleading or at least is difficult to understand.

Reply:
Well I'll delete this note.
What I wanted to say was "please don't miss the last line! It has been added!" so this note is just a transient advisory.

Reply:
So, shall we write that in the release note?